tests/utils/test_visualizations.py

```diff
@@ -369,11 +369,11 @@ def test_plot_skyplot(navdata, state_estimate):
             navdata["x_sv_m",col_idx] = np.nan
 
     # don't save figures
-    fig = viz.plot_skyplot(navdata, state_estimate, save=True)
+    fig = viz.plot_skyplot(navdata.copy(), state_estimate, save=False)
     viz.close_figures(fig)
 
     with pytest.raises(TypeError) as excinfo:
-        viz.plot_skyplot(navdata, state_estimate, save=True, prefix=1)
+        viz.plot_skyplot(navdata.copy(), state_estimate, save=True, prefix=1)
```

Comment:
Why are we copying the navdata here? Asking just out of curiosity

Reply:
because plot_skyplot() adds elevation and azimuth inplace in navdata, but that wouldn't be tested further if it's not a copy